api/proto/common/common.proto

```diff
@@ -2,6 +2,10 @@ syntax = "proto3";
 package common;
 option go_package = "github.com/Layr-Labs/eigenda/api/grpc/common";
 
+// TODO: I generated the comments in this file using copilot. Reviewers, please verify accuracy,
```

Comment:
Please remove

(btw you may just comment on the file to give reviewers notice instead of writing in the code)

Reply:
removed